### ISSUE TITLE
Sometimes alerts are not announced when navigating in the editor with arrow keys

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#203266]
@rperez030, when I test the alerts in the editor - specifically for errors and warnings, sometimes the alerts are not read despite the call succeeding. 

Note that you'll need to enable the audio cue to get the alert, will be fixed / tracked by #203211.

When this happens, the line content is read instead. 

@aiday-mar has reported also having the opposite experience - that is, when navigating an editor with arrow keys, she will hear the alert, for example "error, breakpoint" and not the line content. 

Have you experienced these issues?

**Comment by meganrogge** [comment]
I'm wondering if this is not an issue because you are using a braille device.

**Comment by rperez030** [comment]
It is really an issue in macOS. The reason is likely because alerts cause voiceover to remove the content of the speaking queue as a way to give alerts the highest priority. These should probably be status messages instead of alerts. This shouldn't hit production as it is right now.

**Comment by meganrogge** [comment]
Makes sense, thanks. I will go with `status` instead.

**Comment by rperez030** [comment]
I'll test as soon as it gets merged.

**Comment by meganrogge** [comment]
I am testing and see the `status` message is often not announced now. It's still a better solution because we know it won't override the actual line content.

**Comment by meganrogge** [comment]
separating the issues #203113

**Comment by meganrogge** [comment]
it looks like we might need to do something hacky like this https://stackoverflow.com/questions/58734798/aria-live-region-is-not-reading-out-updates-when-an-element-is-removed-on-chrome

@isidorn, did you encounter similar issues and did you consider doing this?

**Comment by rperez030** [comment]
> I am testing and see the `status` message is similarly not announced after the first call (despite debugging showing that the `status` call does happen). > 

That is interesting. To me it is reading consistently with the audio cue, just overriding the readout of the actual content due to the way in which alerts are handled. In other words, I hear the audio cue and get the alert right after consistently. I'm running macOS sonoma.

**Comment by meganrogge** [comment]
@rperez030, that's great insight. Perhaps this is once again just an issue with my VoiceOver configuration then. Could you pls test in insider's tomorrow, where the `status` will occur instead of `alert`?

**Comment by isidorn** [comment]
@meganrogge I was not encountering issues like this in the past, so I do not have much good context. Sorry

**Comment by meganrogge** [comment]
Closing as an issue with my setup until I'm told otherwise via @rperez030
